framework/src/engine/generator/schemas.ts

```diff
@@ -76,12 +114,30 @@ export const updateStatusRequestSchema = {
 		maxHeightPrevoted: {
 			type: 'integer',
 		},
-		overwrite: {
-			type: 'boolean',
+	},
+};
+
+export interface EstimateSafeStatusRequest {
```

Comment:
According to the estimateSafeStatus definition in the ticket, the request structure is

{ Address: 0x00000, timeShutdown: 222222222222 }

Address seems to me missing from this interface.

Reply:
ah yes, during the implementation, I realized `address` was actually not used in anything. I will update the issue